draft-thomson-webpush-protocol.xml

```diff
@@ -285,11 +290,17 @@
               server <xref target="send"> sends push messages</xref> using this
               resource.
             </t>
+            <t hangText="receipt:">
+              A link relation of type "urn:ietf:params:push:receipt" refers to a
+              resource with read access to delivery receipts.  An application
+              server <xref target="request_receipt"> receives delivery
+              confirmations </xref> for push messages using this resource.
+            </t>
```

Comment:
What do you think about calling this a "receipt subscription"?  That is, it's a type of subscription that receives receipts.  I think that you are already there mostly.

Nit: don't add a space after the `<xref target="...">`.